src/components/About.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+
+const About = () => {
+  return (
+    <div
+      id="about_us"
+      className="grid align-center justify-center bg-cover bg-center bg-no-repeat min-h-screen min-w-full bg-white text-slate-700 text-3xl"
+    >
+      <div className="text-center	leading-normal max-w-auto ">
+        <p className="justify-self-center mt-32 text-4xl mb-3">About Us</p>
+        <p className="mx-52 text-xl text-balance">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Dui sapien
```

Comment:
Just some lorem ipsum. Thomas referred to this as "latin" in one of his old PRs.